### DISCUSSION TITLE
Nugets in `.vcxproj`

### DISCUSSION THREAD

**Discussion by renefritze** [renovatebot/renovate#29032] [Request Help]
### What would you like help with?

Other

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitHub

### Please tell us more about your question or problem

We're pulling in native C++ dependencies via NuGets into `.vcxproj`. Judging from the [documentation](https://docs.renovatebot.com/nuget/#version-support) this simply is not supported. 
We currently use `packages.config` files for each solution and import targets files manually.
Is there a way forward for us to use renovatebot?
Would moving to `PackageReference` and central version management for example help?

**Comment by rarkins** [comment]
I'm not a nuget expert but a reproduction repo would be required if the decision is that something new needs adding
